src/Symfony/Component/Security/Core/CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+6.1
```

Comment:
The feature freeze for 6.1 was at the end of March. While we were still merging some older PRs that were in review, new PRs have to wait for 6.2 (and as 6.1.0-beta1 has been released, even older PRs now need to wait).

Reply:
I can update it all in one go when it's time.  There's no 6.2 branch yet so for the sake of consistency I just used 6.1 everywhere.

Reply:
You can use 6.2 for the sake of consistency. 🙂 

We will retarget your PR as soon as the 6.2 branch is open.